ddtrace/contrib/trace_utils.py

```diff
@@ -175,3 +176,17 @@ def set_http_meta(
 
     if response_headers is not None:
         store_response_headers(dict(response_headers), span, integration_config)
+
+
+def activate_distributed_tracing_context(pin, int_config, request_headers):
```

Comment:
```suggestion
def activate_distributed_tracing_context(tracer, int_config, request_headers):
```

I don't think we use `Pin`s consistently enough throughout the library for the tracer instance (see Pyramid for example). So I think we can just pass a tracer reference directly.